src/components/app.tsx

```diff
@@ -189,39 +216,62 @@ export const App: React.FC<IAppProps<IModelInputState, IModelOutputState, IModel
   const backgroundImage = currentEnvironment?.backgroundImage;
   const leafPackState = LeafPackStates.find((ls) => ls.leafDecomposition === leafDecomposition) || LeafPackStates[0];
 
-  const [trayAnimals, setTrayAnimals] = useState<TrayAnimal[]>([]);
+  const [trayObjects, setTrayObjects] = useState<TrayObject[]>([]);
   const [showTray, setShowTray] = useState(false);
+
   const handleRewind = () => {
     setShowTray(false);
-    setTrayAnimals([]);
+    setTrayObjects([]);
+    setTraySelectionType(undefined);
     rewindSimulation();
   };
 
   const [traySelectionType, setTraySelectionType] = useState<TrayType | undefined>(undefined);
   const handleTrayObjectSelect = (objectType: TrayType) => {
-    if (objectType !== LeafType.leaf) {
+    if (objectType !== LeafType.birch && objectType !== LeafType.oak && objectType !== LeafType.maple) {
       setTraySelectionType(objectType);
     }
   };
   const handleCategorizeAnimal = (trayType: TrayType | undefined, notebookType: TrayType | undefined) => {
     if (trayType === notebookType && trayType !== undefined) {
-      const updatedTrayAnimals = trayAnimals.map(ta => {
-        if (ta.type === trayType) {
-          const collectedTrayAnimal = { ...ta };
+      const updatedTrayObjects = trayObjects.map(to => {
```

Comment:
I initially read `to` as in from/to. Perhaps `obj` might be clearer.

Reply:
Fixed.  I tried to fix a handful of these to make them clearer.